scalatest/src/main/scala/org/scalatest/InsertionOrderSet.scala

```diff
@@ -17,7 +17,7 @@ package org.scalatest
 
 private[scalatest] object InsertionOrderSet {
   def apply[A](elements: List[A]): Set[A] =
-    scala.collection.immutable.SortedSet(elements: _*)(new Ordering[A] {
-      def compare(x: A, y: A): Int = elements.indexOf(x) compare elements.indexOf(y)
-    })
+    scala.collection.immutable.SortedSet(elements: _*)(new Ordering[A | Null] {
+      def compare(x: A | Null, y: A | Null): Int = elements.indexOf(x) compare elements.indexOf(y)
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.